src/server/dflycmd.cc

```diff
@@ -658,7 +658,8 @@ void DflyCmd::StartStableSyncInThread(FlowInfo* flow, ExecutionState* exec_st, E
   DCHECK(shard);
   DCHECK(flow->conn);
 
-  flow->streamer.reset(new JournalStreamer(sf_->journal(), exec_st, JournalStreamer::SendLsn::YES));
+  flow->streamer.reset(
+      new JournalStreamer(sf_->journal(), exec_st, JournalStreamer::SendLsn::YES, true));
```

Comment:
why do you need `is_stable_sync` argument if it's always true?

Reply:
Afaict, RestoreStreamer uses JournalStreamer to replicat data and it expects immediate response back. Data that is sents is less than threshold so it is slow process (i had several test failing). So for RestoreStreamer we don't do this aggregation and behaviour is as before (dispatch data immediately) - while for stable sync replication this is enabled.

Reply:
ah, i missed that